test-helpers/src/main/java/cz/xtf/testhelpers/image/ImageContent.java

```diff
@@ -107,7 +107,8 @@ public PodShell shell() {
 	}
 
 	public List<String> listZipFilesInDir(String path) {
-		return shell.executeWithBash("ls -R1 ~ " + path + " | grep '\\.zip'").getOutputAsList();
+		return shell.executeWithBash("ls -R1 ~ " + path + " | grep '\\.zip'").getOutputAsList()
```

Comment:
`...grep '\\.zip$'...` or `...grep '\\.zip\\$'...` should do the job... we are already filtering results by grep no need to do it again in java. However I'm no sure witch version is correct and whether '$' needs to be escaped.